src/domains/ecommerce-app/env/weu-uat/terraform.tfvars

```diff
@@ -58,3 +58,19 @@ pagopa_vpn_dr = {
     "0.0.0.0",
   ]
 }
+
+idr_subnet_ecommerce_functions = "<TO_BE_DEFINED_>" //"10.1.196.0/24"
+
+ecommerce_function_always_on = false
+
+ecommerce_functions_app_sku = {
+  kind     = "Linux"
+  sku_tier = "Standard"
```

Comment:
```suggestion
  sku_tier = "Basic"
```

Reply:
fixed in 6d763db67451e61f3e7df391de38bc3446039216